services/port_manager/src/main/resources/application.properties

```diff
@@ -28,8 +28,13 @@ microservices.elasticip.service.url=http://localhost:9011
 #####Http Server#####
 server.max-http-header-size=16384
 server.max-initial-line-length=65536
+
+microservices.connectTimeout=300
+microservices.readTimeout=300
```

Comment:
This unit is second, different from yaml file.

Reply:
This is for restful connections (PM TO DPM).

Reply:
Oh I see. Could we make them both in seconds, to be consistent?